packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-amount-mobile.tsx

```diff
@@ -1,11 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import { AMOUNT_MAX_LENGTH, getDecimalPlaces } from '@deriv/shared';
 import { MobileWrapper } from '@deriv/components';
-import { connect } from 'Stores/connect';
+import { connect } from '../../../../../../Stores/connect';
 import { localize } from '@deriv/translations';
 import LabeledQuantityInputMobile from '../../LabeledQuantityInputMobile';
+import { useStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types';
+
+type TAccumulatorsAmountMobile = {
+    amount: ReturnType<typeof useStore>['trade']['amount'];
+    currency: ReturnType<typeof useStore>['trade']['currency'];
+    current_focus: ReturnType<typeof useStore>['ui']['current_focus'];
+    is_nativepicker: boolean;
+    is_single_currency: ReturnType<typeof useStore>['client']['is_single_currency'];
+    onChange: ReturnType<typeof useStore>['trade']['onChange'];
+    setCurrentFocus: ReturnType<typeof useStore>['ui']['setCurrentFocus'];
```

Comment:
@henry-deriv please remove all dependencies from external types for the component https://github.com/binary-com/deriv-app/pull/8268/files#r1170886143